jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParamsDriver.java

```diff
@@ -170,47 +181,66 @@ public static Result run(String inPaths, String pkgName, String outPath, boolean
                       .toString()
                       .equals("Primordial")) {
                 Preconditions.checkNotNull(mtd, "method not found");
-                // Skip methods by looking at bytecode
-                try {
-                  if (CodeScanner.getFieldsRead(mtd).isEmpty()
-                      && CodeScanner.getFieldsWritten(mtd).isEmpty()
-                      && CodeScanner.getCallSites(mtd).isEmpty()) {
-                    continue;
+                DefinitelyDerefedParams analysisDriver = null;
+                String sign = "";
+                // Parameter analysis
+                if (mtd.getNumberOfParameters() > (mtd.isStatic() ? 0 : 1)) {
+                  // Skip methods by looking at bytecode
+                  try {
+                    if (!CodeScanner.getFieldsRead(mtd).isEmpty()
+                        || !CodeScanner.getFieldsWritten(mtd).isEmpty()
+                        || !CodeScanner.getCallSites(mtd).isEmpty()) {
+                      // Make CFG
+                      IR ir =
+                          cache
+                              .getIRFactory()
+                              .makeIR(mtd, Everywhere.EVERYWHERE, options.getSSAOptions());
```

Comment:
Will this and the next case put together mean calling `makeIR` twice on some methods? Would that have a performance impact?